jujugui/static/gui/src/app/components/deployment/panel/content/content.js

```diff
@@ -27,15 +27,17 @@ YUI.add('deployment-panel-content', function() {
     },
 
     render: function() {
+      var props = this.props;
+      var className = props.className || '';
```

Comment:
Or in fact maybe not need this it all if you just apply a wrapping class to the component this is used with (see my other component where this is used).